test/gw-ci/create_exp.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
-expyaml="$1"
-export pslot="$2"
+
 HOMEgfs="$3"
-export RUNTESTS="$4"/${pslot}
-export SLURM_ACCOUNT="da-cpu"
 
 # Get ICSDIR_ROOT
 source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ci/platforms/config.${MACHINE_ID}"
 
+# Arguments for the exp setup
+expyaml="$1"
+export pslot="$2"
+export RUNTESTS="$4"/${pslot}
+export SLURM_ACCOUNT="da-cpu"
```

Comment:
so I'm guessing this is here to overwrite something in the CI config, but why are the $1 and $2 things moved down here?